frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignDrawer/EditEndPageDrawer/EditEndPage.tsx

```diff
@@ -133,5 +132,5 @@ export const EndPageBuilderInput = ({
 export const EditEndPage = (): JSX.Element => {
   const { data: form } = useAdminForm()
 
-  return <>{form ? <EndPageBuilderInput endPage={form.endPage} /> : null}</>
+  return form ? <EndPageBuilderInput endPage={form.endPage} /> : <></>
```

Comment:
nit: can probably return `null` in this case, we just need to update the return type of this component

```tsx
export const EditEndPage = (): JSX.Element | null => {
```

Reply:
errrr that would have other effects upstream i think? (in `BuilderAndDesignContent`) should we do that in this PR haha

Reply:
won't have other effects. can you try? should work as per normal

Reply:
oop you are right hahah